### PR TITLE
ETS-1280 fix scalaVersion command

### PR DESCRIPTION
**Pull request by AndyKirsch**
@AudaxHealthInc/acceleration 

Was a shot in the dark that happened to fix the problem. Still don't understand why.

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -6,6 +6,6 @@ resolvers += Resolver.url(
   url("https://artifacts.werally.in/artifactory/ivy-plugins-release")
 )(Resolver.ivyStylePatterns)
 
-addSbtPlugin("com.rallyhealth" %% "rally-versioning" % "latest.release")
+addSbtPlugin("com.rallyhealth" %% "rally-versioning" % "1.3.0")
```
Comment (kapkaev) [inline project/plugins.sbt:9]:
```suggestion
addSbtPlugin("com.rallyhealth" %% "rally-versioning" % "1.10.1")
```
Latest release version https://github.com/AudaxHealthInc/rally-versioning/releases/tag/v1.10.1

Comment (AndyKirsch) [reply project/plugins.sbt:9]: That was https://github.com/AudaxHealthInc/akka-construct/pull/6/commits/99323a70e4405e922562ae3071f1814e3caed83b which had an issue with the version of git on the build server so I backed up to an older version

**Comment by AndyKirsch** [comment]
@usufruct99 since permissions here are wonky

**Comment by AndyKirsch** [comment]
Need a (squash) merge from someone who can

**Pull request merged**